### PR TITLE
Initial PR

### PR DESCRIPTION
**Pull request by vadimshvetsov**
Initial PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# eslint-config-qlean
-JavaScript Qlean Style - EsLint Shareable Config
+# Eslint Config
+
+JavaScript web and React Native Qlean style shareable configs
```
Comment (alextewpin) [inline README.md]: I suggest to cut the title, write something like "Qlean extensions for Airbnb config" and add links to actual configs readmes.

File: packages/eslint-config-qlean-react-native/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# eslint-config-qlean-react-native
+
+This package provides Qlean's `.eslintrc` as an extensible shared config for linting React Native codebase.
```
Comment (alextewpin) [inline packages/eslint-config-qlean-react-native/README.md]: 1. It's airbnb config copypaste
2. It's not true, cuz we extending airbnb config

File: packages/eslint-config-qlean-react-native/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## Installation
+
+1. Install the correct versions of each package, which are listed by the command:
+```sh
+npm info "eslint-config-qlean-react-native" peerDependencies
+```
+2. Install the latest version of `eslint-config-qlean-react-native` package with following command:
+```sh
+npm install --save-dev eslint-config-qlean-react-native
+```
```
Comment (alextewpin) [inline packages/eslint-config-qlean-react-native/README.md]: Comments are redundant, `--save-dev` too. Just write one actual command, like:
`$ npm install eslint-config-qlean-react-native and-all-peer-deps`

File: packages/eslint-config-qlean-react-native/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Usage
+
+1. Add `"extends": "qlean-react-native"` to your .eslintrc
```
Comment (alextewpin) [inline packages/eslint-config-qlean-react-native/README.md]: What the point of numeration when there is one item in list?

File: packages/eslint-config-qlean/package.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-qlean",
+  "version": "1.0.0",
+  "description": "JavaScript Qlean Style - EsLint Shareable Config",
```
Comment (alextewpin) [inline packages/eslint-config-qlean/package.json]: ESLint is obvs for JS. And what exactly "shareable" means here?

File: packages/eslint-config-qlean/package.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  "main": "index.js",
+  "peerDependencies": {
+    "babel-eslint": ">= 8",
+    "eslint": ">= 4",
+    "eslint-config-airbnb": ">= 15"
```
Comment (alextewpin) [inline packages/eslint-config-qlean/package.json]: Why `eslint-config-airbnb` is peer? Should be dependency.

File: packages/eslint-config-qlean/package.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-qlean",
+  "version": "1.0.0",
+  "description": "JavaScript Qlean Style - EsLint Shareable Config",
+  "main": "index.js",
```
Comment (alextewpin) [inline packages/eslint-config-qlean/package.json]: Do we even have this file?

File: packages/eslint-config-qlean-react-native/README.md (hunk 2 above)
Comment (vadimshvetsov) [reply packages/eslint-config-qlean-react-native/README.md]: @alextewpin It is true. We use our config as extensible shared config and it doesn't matter that we extends Airbnb or whatever. Any suggestions?

File: packages/eslint-config-qlean-react-native/README.md (hunk 3 above)
Comment (vadimshvetsov) [reply packages/eslint-config-qlean-react-native/README.md]: @alextewpin Why `--save-dev` is redundant?

Comment (alextewpin) [reply packages/eslint-config-qlean-react-native/README.md]: My bad, `--save` is.

File: packages/eslint-config-qlean-react-native/README.md (hunk 2 above)
Comment (alextewpin) [reply packages/eslint-config-qlean-react-native/README.md]: I think it's fine to just cut this paragraph and leave this kind of blablabla stuff to main `README`, `eslint-config-qlean-react-native` is self-explanatory enough.

File: packages/eslint-config-qlean/package.json (hunk 5 above)
Comment (vadimshvetsov) [reply packages/eslint-config-qlean/package.json]: @alextewpin [Shareable configs](https://eslint.org/docs/developer-guide/shareable-configs)

File: packages/eslint-config-qlean/package.json (hunk 7 above)
Comment (vadimshvetsov) [reply packages/eslint-config-qlean/package.json]: @alextewpin yeah, shareable config is there

File: packages/eslint-config-qlean/README.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+2. Install the latest version of `eslint-config-qlean` package with following command:
+```sh
+npm install --save-dev eslint-config-qlean
+```
+
+## Usage
+
+1. Add `"extends": "qlean"` to your .eslintrc
+
+## Motivation for disabling rules
```
[the remark this answers is not in the document]
Comment (vadimshvetsov) [reply packages/eslint-config-qlean/README.md]: @alextewpin could we remove motivation section? no one reacts to my call

File: packages/eslint-config-qlean/index.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    "airbnb-base",
+    "airbnb-base/rules/strict",
+    "airbnb/rules/react"
+  ],
+  plugins: [
+    "no-loops"
+  ],
+  "env": {
+    "browser": true,
+    "node": true
```
Comment (pavlov-andrey) [inline packages/eslint-config-qlean/index.js]: еще бы
```js
"jest": true
```

**Pull request merged**